src/Test/Utilities/Portable/CompilerTraitAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I was suggesting one use of the attribute and using a named constant for the assemblyName parameter. That named constant could have different values in the two assemblies.